core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/expressions.scala

```diff
@@ -238,3 +238,7 @@ case class Variable(name: String) extends LeafExpression {
 case class SchemaReference(columnName: Expression) extends Unary(columnName) {
   override def dataType: DataType = UnresolvedType
 }
+
+case class JoinMarkExpression(value: Expression) extends LeafExpression {
```

Comment:
how is it named in catalyst?

Reply:
Its work in progress and a draft PR, As a practice I commit the dev code on regular basis.